Makefile.toml

```diff
@@ -628,3 +628,82 @@ script = [
     end
     """
 ]
+
+
+
+#  ##     ## ####    ######## ########  ######  ######## 
+#  ##     ##  ##        ##    ##       ##    ##    ##    
+#  ##     ##  ##        ##    ##       ##          ##    
+#  ##     ##  ##        ##    ######    ######     ##    
+#  ##     ##  ##        ##    ##             ##    ##    
+#  ##     ##  ##        ##    ##       ##    ##    ##    
+#   #######  ####       ##    ########  ######     ##    
+
+[tasks.test-server]
+# condition = { platforms = ["mac"], env_true = ["RELEASE"] }
+script_runner =  "@duckscript"
+script = [
+    """
+    os = os_family
+    cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/util/test_server/
+    if eq ${os} "linux"
+        exec docker-compose up -d
+    else
+        exec vagrant up --provision
+    endif
+    """
+]
+
+[tasks.ui-test-manager]
+private = true
```

Comment:
make this public

```suggestion
```

Reply:
Unfortunately this still doesn't resolve the error I am facing....